huaweicloud/services/ucs/resource_huaweicloud_ucs_policy.go

```diff
@@ -0,0 +1,354 @@
+// ---------------------------------------------------------------
+// *** AUTO GENERATED CODE ***
+// @Product UCS
+// ---------------------------------------------------------------
+
+package ucs
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/jmespath/go-jmespath"
+
+	"github.com/chnsz/golangsdk"
+	"github.com/chnsz/golangsdk/pagination"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/common"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+)
+
+func ResourcePolicy() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourcePolicyCreate,
+		UpdateContext: resourcePolicyUpdate,
+		ReadContext:   resourcePolicyRead,
+		DeleteContext: resourcePolicyDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"region": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ForceNew: true,
+			},
+			"name": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: `Specifies the name of the UCS policy.`,
+			},
+			"iam_user_ids": {
+				Type:        schema.TypeList,
+				Elem:        &schema.Schema{Type: schema.TypeString},
+				Required:    true,
+				Description: `Specifies the list of iam user IDs to associate to the policy.`,
+			},
+			"type": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: `Specifies the type of the UCS policy.`,
+			},
+			"description": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Computed:    true,
```

Comment:
```suggestion
```
This parameter is support left blank via update function during API .

Reply:
fixed